martin/src/srv/tiles.rs

```diff
@@ -77,7 +82,7 @@ impl<'a> DynTileSource<'a> {
         source_ids: &str,
         zoom: Option<u8>,
         query: &'a str,
-        accept_enc: Option<AcceptEncoding>,
+        (accept_enc, if_none_match): (Option<AcceptEncoding>, Option<IfNoneMatch>),
```

Comment:
this is done like this to not make clippy angry that these are defintively too many arguments.

Reply:
let's just add the `#[expect(...)]` here - much better than to be confused about the meaning of the code